contrib/config/get_luet_root.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
+if [ $(id -u) -ne 0 ]
```

Comment:
That's good!

wondering if we can try to reexec ourselves and spare users to re-run the command at this point with something like: 

```
[ $(id -u) -eq 0 ] || exec sudo $0 $@
```

but unsure if works with curl, since we pipe this